keyboards/planck/rev1/config.h

```diff
@@ -0,0 +1,8 @@
+#ifndef REV1_CONFIG_H
+#define REV1_CONFIG_H
+
+#include "config_common.h"
```

Comment:
This line is not actually needed, and should be removed. 

Specifically, this should be included in the keyboard's `config.h` file, and is redundant. 

```suggestion
```